pom.xml

```diff
@@ -5,7 +5,7 @@
   <groupId>com.github.os72</groupId>
   <artifactId>protoc-jar-maven-plugin</artifactId>
   <packaging>maven-plugin</packaging>
-  <version>3.1.0.2</version>
+  <version>3.1.0.2-tet-SNAPSHOT</version>
```

Comment:
Could you not make it a -SNAPSHOT version, so we don't have to keep downloading it every time we build?